### ISSUE TITLE
Genbank location parser breaks on parsing this location.

### ISSUE THREAD

**Issue by TimothyStiles** [TimothyStiles/poly#384]
## Describe the bug
`Genbank.Read` panics when trying to read this specific location line

```
     misc_RNA        join(complement(5306942..5307394),
                     complement(5304401..5305029),complement(5303328..5303393),
                     complement(5301928..5302004))
```
### Severity of the bug
Keeps us from parsing refseq human genome

### Steps to reproduce
*Steps to reproduce the behavior:*
Will update with better steps but you should be able to create a new genbank file with this location and run it through `genbank.Read` to reproduce.

### Expected behavior
Will not panic and be able to parse location string effectively.

### Screenshots

![Screen Shot 2023-10-06 at 7 14 50 PM](https://github.com/TimothyStiles/poly/assets/7042260/aa55ec6c-cb15-4dd1-9a3f-44effdf889bd)

### Client information
 - OS: Mac OSX 0.15.7
 - Golang version: `1.21`
 - `poly` version `v0.27.1`

**Comment by TimothyStiles** [comment]
fixed as of #385

**Comment by TimothyStiles** [comment]
fixed as of #385
